Makefile

```diff
@@ -7,19 +7,19 @@ all: build run
 	
 build:
 	meson compile -C build -v
+	cd build && ninja
```

Comment:
Ubuntu 20 in Docker installs `meson` 0.53 which requires building code.

![image](https://user-images.githubusercontent.com/282605/106468370-7f6b0180-64a6-11eb-868b-6acd5585fd3e.png)